src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
```

Comment:
https://github.com/JasonTaranenko/jv-oop/blob/homeWork/checklist.md#please-dont-add-redundant-empty-lines-to-your-code